libraries/core_libs/network/src/tarcap/packets_handlers/dag_sync_packet_handler.cpp

```diff
@@ -55,12 +79,10 @@ void DagSyncPacketHandler::process(const PacketData& packet_data, const std::sha
     dag_blk_mgr_->insertBroadcastedBlock(std::move(block));
   }
 
-  if (missing_blks.size() > 0) {
-    requestDagBlocks(packet_data.from_node_id_, missing_blks, DagSyncRequestType::MissingHashes);
-  }
-  syncing_state_->set_dag_syncing(false);
+  peer->peer_dag_synced = true;
+  peer->peer_dag_syncing = false;
```

Comment:
so as dag_syncing is moved out from syncing_state, once we are pbft synced (but not yet dag synced), received dag blocks from other peers will probably fail because we are not dag synced yet, is that right and are we ok with it ?

Reply:
+ didnt you changed testnet that we are sending dag blocks in batches of 100 ? In such case this would not work properly ?

Reply:
The batch of 100 is removed from testnet as well. This was there only because I originally did not realize what was causing the slow processing. And yes we are ok with received dag blocks failing with tips/pivot missing until dag_synced is true, once dag_synced is true for a peer any missing tip/pivot should indicate a malicious node. Currently I disconnect such a node but we should have some blacklisting as well not to reconnect to it again